### PR TITLE
W 11673762 processautomation editdeletedeactivatapplications se

### PR DESCRIPTION
**Pull request by sebermuley**
Added edit, deletion and (de)activation to creation of application and grouped them under Manage Applications. Added anchors for links to assignment of applications to bots and processes.

### CODE REVIEW

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 1]
```diff
@@ -40,9 +43,15 @@ Create categories to group processes. Assign a process to a category during eval
 . Enter a name in the *Create a category* window.
 . Click *OK*.
 
-=== List Applications
+=== Manage Applications
+
+In the view *Process Applications*, list all applications required for the execution of processes. In the view *Process Management*, assign required applications to processes at process creation or phase transition. In the module *Bot Management*, assign applications installed on a computer a bot is installed on to that bot.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:48]: I had to reread the last sentence a couple of times to understand it. Could it be simplified to just: 

In the module *Bot Management*, assign applications to a bot installed on a computer.

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc [hunk 2]
```diff
@@ -40,9 +43,15 @@ Create categories to group processes. Assign a process to a category during eval
...
+
+Applications are not available during down-times or while deactivated. A process can be deployed on a bot which has assigned the required processes. Processes whose applications are currently down are executed to completion and are not restarted until the downtime or deactivation period has ended.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:50]: General comment: If possible, avoid passive voice.

**Review by Cristian-Venticinque — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc (hunk 1 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc:48]: Yes, it should be simplified, but no, that is too litlle information:)

**Comment by sebermuley** [comment]
Please re-review. Thank you.

**Review by jeberhardST — DISMISSED** [review]

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**